lib/index.js

```diff
@@ -279,6 +279,8 @@ exports.parseEprints = BBPromise.method(function(chtml){
 exports.parseGeneral = BBPromise.method(function(chtml){
 
 	var clutteredMeta = {
+		appleTouchIcons: chtml('link[rel=apple-touch-icon i]').map(function(i, e) { return { url: e.attribs.href, size: e.attribs.sizes } }).get(), //apple-touch-icon <link rel="apple-touch-icon" href="" sizes="" type="">
```

Comment:
adds undefined values if undefined (i.e. sizes)

Reply:
Hi, I'm trying to fix this but I'm not sure what it should do when it's undefined.

Reply:
If size is undefined, then you should just not add it to the resulting object. So right now, say if size is undefined, this object is returned:

{ url: "http://example.com",
size: undefined}

and we want

{ url: "http://example.com" }

Reply:
You might actually find it helpful to add tests for the apple touch icons first. You can add the html for the icons to this file: https://github.com/wikimedia/html-metadata/blob/master/test/static/turtle_article.html

And the corresponding tests for that html file are in https://github.com/wikimedia/html-metadata/blob/master/test/static.js